README.md

```diff
@@ -6,10 +6,91 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an service for operating and monitoring electrical
+components and sensors within a microgrid. It offers various functionalities
+related to microgrid management, including metadata retrieval, component
+listing, setting power levels, and controlling the state of components. The API
+is structured around the concept of a microgrid, which is a localized energy
+system composed of various interconnected components and auxiliary sensors.
+Microgrids are typically connected to the main grid, but can also operate
+independently, in which case they are referred to as _islanded_ microgrids.
 
-This repository provides the core gRPC and protobuf spec together with
-support for generating dedicated API packages for various languages.
+The API supports different _categories_ of electrical components and sensors.
+The list of supported electrical component categories cam be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.4.0/proto/frequenz/api/common/v1/components.proto#L13-L72).
+The list of supported sensor categories cam be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/83e96a9cb3fd88db06c47d29be54da435fc648e3/proto/frequenz/api/common/v1/sensors.proto#L13-L38).
+Note that each of these categories could have their own _types_, e.g., inverters
+could belong to the type "battery" or "PV". This information is stored in the
+metadata of each component and sensor, and is exposed by the API.
```

Comment:
Suggestion from me and my deep knowledgeable friend.

Note, that each of these categories could have its own types; for example, inverters could belong to the types 'battery' or 'PV.' This information is stored in the metadata of each component and sensor and is exposed by the API.

Reply:
>inverters could belong to the types 'battery' or 'PV.' 

incorrect English :)

Reply:
the actual changes that are the commas btw :)